site/content/docs/5.3/examples/footers/index.html

```diff
@@ -13,12 +13,12 @@ <h1 class="visually-hidden">Footers Boosted example</h1>
 
   <div class="b-example-divider"></div>
 
-  {{< orange-footer mandatory=true id="1" >}}
+  {{< orange-footer mandatory=true in_examples=true id="1" >}}
   {{< /orange-footer >}}
 
   <div class="b-example-divider"></div>
 
-  {{< orange-footer nav=true mandatory=true id="2" >}}
+  {{< orange-footer nav=true mandatory=true in_examples=true id="2" >}}
```

Comment:
IMO we shouldn't set the active links in the footers examples because there's no context justifying this rendering here, but rather in the Footers documentation page, maybe in a specific section where it is explained why it is helpful to do so.
Something explaining that this helps accessibility, why, and that users can just add the `.active` class on whatever link element in the footer to have this behavior (and show it for 1 type of footer).
And so, except for the Form example and this section, don't show this behavior when there's no appropriate context.

I can help with a commit if needed.